docs/_layouts/page.html

```diff
@@ -5,6 +5,9 @@
 <div class="post-header">
     <a id="{{page.title}}"></a>
     <h1 class="post-title-main">{{ page.title }}</h1>
+    {% if page.nb_path %}
```

Comment:
@jph00 this line of code prevents non-notebook documents from getting a Colab badge